ddp-workspace/projects/ddp-sdk/src/lib/services/internationalization/ngxTranslate.service.ts

```diff
@@ -1,17 +1,30 @@
 import { Injectable } from '@angular/core';
 import { TranslateService } from '@ngx-translate/core';
-import { Observable, of } from 'rxjs';
-import { mergeMap, merge } from 'rxjs/operators';
+import { merge, Observable, of } from 'rxjs';
+import { mergeMap } from 'rxjs/operators';
 
 @Injectable()
 export class NGXTranslateService {
     constructor(private translate: TranslateService) { }
 
-    public getTranslation(word: string | Array<string>): Observable<object | string> {
-        return of(null).pipe(
-            merge(this.translate.onLangChange),
-            merge(this.translate.onDefaultLangChange),
-            mergeMap(() => this.translate.get(word))
+    public getTranslation(word: string, interpolateParams?: object): Observable<string>;
+    public getTranslation(words: Array<string>, interpolateParams?: object): Observable<object>;
+
```

Comment:
Defining it this way makes it easier to determine types for the users of the method: if you pass a string, you are getting a string. If you pass an array, you are getting an object with the lookup keys as properties and the values as the translated strings.